assets/js/services/metadata.js

```diff
@@ -1,3 +1,82 @@
+export const CONTROLLED_METADATA = [
```

Comment:
Looks like `CONTROLLED_METADATA` and `DESCRIPTIVE_METADATA.controlledTerms` are tracking the same thing and both being used?  This may lead to confusion moving forward.   Can we consolidate into using one or the other?

Reply:
So the idea was to get rid of `DESCRIPTIVE_METADATA` as a whole when the tests in localstorage are fixed. Looks like I missed removing it from About component.New commit up in a few.